ParcelDistributionCenter/ParcelDistributionCenter.Model/Entites/Package.cs

```diff
@@ -10,9 +10,8 @@ public class Package : Entity
         // TODO: Think if nullable values are needed
         public Courier? Courier { get; set; }
 
-        [NotMapped]
```

Comment:
Dlaczego nie NotMapped i usunięta ta właściwość? Zrobione przez mnie (podobnie jak wyżej) do mapowania tych Idków, które są potrzebne do Seeda

Reply:
Potrzebuję tego parametru do działania funkcjonalności dalszych. 
Wydaje mi się, że jest to również klucz łączący Courie-Package.